assembl/static2/css/components/announcement.scss

```diff
@@ -0,0 +1,83 @@
+.announcement {
+  text-align: center;
+
+  .announcement-title {
+    .announcement-title-text {
+      margin-top: 0.5em;
+    }
+    text-align: left;
+  }
+
+  .announcement-statistics {
+    display: inline-block;
+    text-align: center;
+    margin-top: 1.5em;
+    margin-bottom: 3em;
+    position: relative;
+    left: 1em;
+    @media (max-width: 767px) {
+      margin-top: 0.1em;
+      left: 0;
+    }
+
+    .announcement-doughnut {
+      width: 145px;
+      height: 145px;
+
+      .superpose-label {
+        top: calc(50% - 2em);
+        line-height: normal;
+
+        .doughnut-label-count {
+          font-size: $font-size-l;
+        }
+
+        .doughnut-label-text {
+          font-family: latoRegular;
+          font-size: $font-size-xs;
+          letter-spacing: 0.08em;
+        }
+      }
+    }
+
+    .announcement-numbers {
+      font-size: 12px;
+      margin-bottom: 15%;
+      color: $first-color;
+
+      span::before {
+        font-size: 14px;
+      }
+    }
+  }
+
+  .announcement-video {
+
+    .video-section {
+
+      .col-sm-8 {
```

Comment:
Don't override a bootstrap class. You have to create your own class.

Reply:
see https://github.com/assembl/assembl/pull/161#discussion_r138108699